pkg/api/copy_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api_test
+
+import (
+	"math/rand"
+	"reflect"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	apitesting "github.com/GoogleCloudPlatform/kubernetes/pkg/api/testing"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/conversion"
+)
+
+func TestDeepCopyApiObjects(t *testing.T) {
+	f := apitesting.FuzzerFor(t, "", rand.NewSource(rand.Int63()))
+	for kind := range api.Scheme.KnownTypes("") {
+		item, err := api.Scheme.New("", "Pod")
```

Comment:
Thanks for catching that. I also added a loop, but I'm unsure what you mean by external versions as well. I'm a little new to this project.